conbench/tests/migrations/test_1fed559406c5_recalculate_distribution3.py

```diff
@@ -1,245 +0,0 @@
-import decimal
```

Comment:
Mostly out of curiosity: this clean up of these tests/migrations/... isn't related to the rename, just cleaning up stuff that we should have cleaned up before, yeah?

Reply:
We should have removed these tests once we were done with migrations.